### PR TITLE
Update Objects.md

### PR DESCRIPTION
**Pull request by brmbrmcar**
Add a few things to it.

### CODE REVIEW

File: Objects.md [hunk 1]
```diff
@@ -1,3 +1,46 @@
...
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/bonus_block/infoblock.png?raw=true "Info Block")
+
+Info Blocks can give valuable information about the level ahead. They can be hit from below or butt-jumped from on top. 
+
+### Trampolines
+
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/trampoline/trampoline1-0.png?raw=true "Portable Trampoline")
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/trampoline/trampoline2-0.png?raw=true "Fixed Trampoline")
+
+Trampolines are available fixed and portable. They can be jumped on to help Tux reach new heights. Portable ones can be moved with the action key. There are also ones that rust and break after a few jumps.
```
Comment (tobbi) [inline Objects.md]: Add something about pressing Shift key for extra heights.

Comment (Karkus476) [inline Objects.md]: That's a thing? You mean action, right?

Comment (tobbi) [inline Objects.md]: I think I mean the action key. Hah. I am getting confused with my own key setup here.

Comment (brmbrmcar) [inline Objects.md]: It doesn't work for me. I don't know what you are on about.

File: Objects.md [hunk 2]
```diff
@@ -4,28 +4,43 @@ Objects can be found throughout the levels in SuperTux.
...
+
+Info Blocks can give valuable information about the level ahead. They can be hit from below or butt-jumped from on top. 
 
 ### Trampolines
 
-Trampolines are available fixed and portable. They can be jumped on to help Tux reach new heights. Portable ones can be moved with the action key. There are also ones that rust and break after a few jumps. ![alt text](https://github.com/SuperTux/data/blob/master/images/objects/trampoline/trampoline1-0.png?raw=true "Portable Trampoline")![alt text](https://github.com/SuperTux/data/blob/master/images/objects/trampoline/trampoline2-0.png?raw=true "Fixed Trampoline")
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/trampoline/trampoline1-0.png?raw=true "Portable Trampoline")
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/trampoline/trampoline2-0.png?raw=true "Fixed Trampoline")
+
+Trampolines are available fixed and portable. They can be jumped on to help Tux reach new heights. Portable ones can be moved with the action key. There are also ones that rust and break after a few jumps.
```
Comment (Karkus476) [inline Objects.md]: Show which is which (fixed, portable). Also include something about Lanterns... Change "moved" to "carried" or "picked up"

Comment (Karkus476) [inline Objects.md]: A picture of the rusty trampoline wouldn't do any harm...

Comment (Karkus476) [inline Objects.md]: Also: Rocks?

Comment (brmbrmcar) [inline Objects.md]: OK. I thought that most portable things should be in Items.md.

File: Objects.md [hunk 3]
```diff
@@ -1,3 +1,59 @@
...
+
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/rock/rock.png?raw=true "Rock")
+
+Rocks can be carried with the action key. They can allow Tux to get over new obstacles.
+
+### Lanterns
+
+![alt text](https://github.com/SuperTux/data/blob/master/images/objects/lantern/lantern-1.png?raw=true "Lantern")
+
+Lanterns can be carried like rocks. However, they are alsoa good light source and can make some blocks of the same colour solid.
```
Comment (tobbi) [inline Objects.md]: alsoa -> also a

Comment (Karkus476) [inline Objects.md]: Actually, a little expansion on the topic would be good.

Comment (brmbrmcar) [inline Objects.md]: good spot

**Comment by maxteufel** [comment]
Can you actually use something useful, instead of "alt text"? It's supposed to be displayed when the graphic isn't yet loaded, or for disabled users.

**Comment by brmbrmcar** [comment]
What else am I meant to do?

**Comment by maxteufel** [comment]
@brmbrmcar Use descriptive text.

**Comment by tobbi** [comment]
After you addressed the comments by max, I am good with merging this.

**Pull request merged**